DevCGlobalDirectory.md

```diff
@@ -168,6 +168,17 @@
   - Github: 
 </details>
 
+<details>
+  <summary>
+    <b>DevC Kathmandu</b>
+  </summary>
+  
+  - Lead(s):
+    - Chandan Thakur
```

Comment:
Thanks for this amazing PR!

Can you please add the GitHub profile link of this lead, please?

Reply:
Sure thing. I will add it soon after confirming it with the lead :)

Reply:
@BolajiAyodeji  Done :)